pkg/Makefile

```diff
@@ -0,0 +1 @@
+include ../../Makefile.Common
```

Comment:
This needs to be in pkg/translator/jaeger, and needs 3 `..` because of that

Reply:
oh yeah, makes sense